src/datasources/cache/__tests__/fake.cache.service.spec.ts

```diff
@@ -49,4 +50,52 @@ describe('FakeCacheService', () => {
 
     expect(target.keyCount()).toBe(0);
   });
+
+  it('deletes keys by pattern', async () => {
+    const prefix = faker.datatype.uuid();
+
+    // insert 5 items matching the pattern
+    await Promise.all(
+      range(5).map(async () =>
```

Comment:
I don't think that you need the mapper function to be `async`. The mapper returns an async function which is then consumed by the `Promise.all`

Reply:
Changed in [54cf021](https://github.com/5afe/safe-client-gateway-nest/pull/288/commits/54cf021cea7b64d7e4246ae066d3c671209e3dde)